src/components/navbar.js

```diff
@@ -0,0 +1,20 @@
+import { useSelector } from 'react-redux';
+import { NavLink } from 'react-router-dom';
+
+function Navbar() {
+  const currencySlice = useSelector((state) => state.currency);
+  return (
+    <nav data-testid="nav-test">
+      <NavLink to="/">
+        <li id="back" />
+      </NavLink>
+      <p>{currencySlice.curCountryCode || 'All country currencies'}</p>
+      <ul>
+        <li id="mic" />
+        <li id="settings" />
+      </ul>
+    </nav>
```

Comment:
- [ ] Good job implementing a very beautiful project using React 👍 . As you required in the guidelines concerning the style, you should follow the [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)) . However, you didn't implement the `hamburger` button 😄  on the left of the navbar. Kindly add it 👍 

### Your design

![Capture](https://github.com/hmunish/react-metrics/assets/80458199/82d1d94c-0c65-4795-9051-1718472c15eb)

### Nelson's design

![Capture1](https://github.com/hmunish/react-metrics/assets/80458199/2d7a94bc-8408-45fa-ba9d-abd82aa84bfe)